app/tests/integration/nimbus/test_mobile_integration.py

```diff
@@ -0,0 +1,158 @@
+import json
+import os
+import time
+import uuid
+from pathlib import Path
+
+import pytest
+import requests
+from nimbus.models.base_app_context_dataclass import BaseAppContextDataClass
+from nimbus.utils import helpers
+
+
+nimbus = pytest.importorskip('nimbus_rust')
+
+def locale_number_loader(locales=None):
+    locale_list = []
+    path = Path().resolve()
+    path = str(path)
+    path = path.strip("/tests/integration/nimbus")
+    path = f"/{path}/experimenter/base/fixtures/locales.json"
+    with open(path) as file:
+        data = json.loads(file.read())
+        for locale in locales:

```

Comment:
no, it will error. I'd like it to fail hard here as if we don't load any locales then we don't have any app contexts and creating the nimbus object will fail.